WEBAPI.Infrastructure/Service/ItemService.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WEBAPI.Application.Interfaces;
+using WEBAPI.Domain.Entities;
+using WEBAPI.Domain.Helpers;
+using WEBAPI.Domain.Models;
+using WEBAPI.Infrastructure.DB;
+
+namespace WEBAPI.Infrastructure.Service
+{
+    public class ItemService : IItemService
+    {
+
+        ApplicationDbContext AppDb;
+
+        private readonly AppSettings _appSettings;
+
+        public ItemService(IOptions<AppSettings> appSettings)
+        {
+            _appSettings = appSettings.Value;
+            AppDb = new ApplicationDbContext(_appSettings.ConnectionString);
+          
+        }
+
+        public IEnumerable<Item> GetAllItems()
+        {
+            string query = "select * from Items";
```

Comment:
please consider only those columns which are required avoid using * in select

Reply:
consider having some pagination to  improve performance

Reply:
Resolved.